src/components/WorkspaceDashboardTaskRunsCard.vue

```diff
@@ -11,7 +11,7 @@
     </div>
 
     <div class="workspace-dashboard-task-runs-card__chart-container">
-      <LineChart :data="taskRunsChartData.failed" :options="{ maxValue: maxFailedValue }" class="workspace-dashboard-task-runs-card__chart workspace-dashboard-task-runs-card__chart--failed" />
+      <LineChart v-if="taskRunsChartData.hasFailed" :data="taskRunsChartData.failed" :options="{ maxValue: maxFailedValue }" class="workspace-dashboard-task-runs-card__chart workspace-dashboard-task-runs-card__chart--failed" />
```

Comment:
Nit: you don't need this new boolean if you just check that failed > 0 same as with the count statistic above.

Reply:
ah danget, will update